src/core/engines/Cesium/Feature/Resource/utils.ts

```diff
@@ -0,0 +1,245 @@
+  const property = entity[propertyName];
+  if (!property) {
+    return;
+  }
+
+  const tag = getTag(entity);
+
+  // Backup original properties
+  overrideOriginalProperties(
+    entity,
+    tag,
+    propertyName,
+    Object.keys(propertyMap).reduce((r, k) => {
+      r[k] = (property as any)[k];
+      return r;
+    }, {} as Record<string, any>),
+  );
+
+  Object.entries(propertyMap).forEach(([entityPropertyKey, appearancePropertyKey]) => {
+    const appearanceKeyName = appearancePropertyKey.name;
+    const appearanceKeyType = appearancePropertyKey.type as AppearancePropertyKeyType;
+
+    let value = (computedFeature?.[appearanceName] as any)?.[appearanceKeyName];
+    switch (appearanceKeyType) {
+      case "color":
+        value = toColor(value);
+        break;
+      case "shadows":
+        value = shadowMode(value);
+        break;
+      case "heightReference":
+        value = heightReference(value);
+    }
+
+    (property as any)[entityPropertyKey] = value ?? (property as any)[entityPropertyKey];
+  });
+}
+
+export const attachStyle = (
+  dataSource: CesiumKmlDataSource | CesiumCzmlDataSource | CesiumGeoJsonDataSource,
+  appearances: (keyof AppearanceTypes)[] | undefined,
+  layer: ComputedLayer | undefined,
+  evalFeature: EvalFeature,
+) => {
+  if (!layer) {
+    return;
+  }
+  dataSource?.entities.values.forEach(entity => {
+    appearances?.forEach(appearance => {
+      if (layer?.layer.type == "simple" && !layer?.layer?.[appearance]) {
+        return;
+      }
+
+      if (appearance === "marker") {
+        const position = entity.position?.getValue(JulianDate.now());
+        const coordinates = [position?.x ?? 0, position?.y ?? 0, position?.z ?? 0];
+        const feature: Feature = {
+          type: "feature",
+          id: entity.id,
+          geometry: {
+            type: "Point",
+            coordinates,
+          },
+          properties: entity.properties?.getValue(JulianDate.now()) || {},
+          range: {
+            x: coordinates[0],
+            y: coordinates[1],
+            z: coordinates[2],
+          },
+        };
+        const computedFeature = evalFeature(layer.layer, feature);
+        attachProperties(entity, computedFeature, ["marker", "point"], {
+          pixelSize: {
+            name: "pointSize",
+          },
+          color: {
+            name: "pointColor",
+            type: "color",
+          },
+          outlineColor: {
+            name: "pointOutlineColor",
+            type: "color",
+          },
+          outlineWidth: {
+            name: "pointOutlineWidth",
+          },
+          heightReference: {
+            name: "heightReference",
+            type: "heightReference",
+          },
+        });
+      }
+
+      if (appearance === "polyline") {
+        const entityPosition = entity.position?.getValue(JulianDate.now());
+        const positions = entity.polyline?.positions?.getValue(JulianDate.now()) as Cartesian3[];
```

Comment:
```suggestion
        const positions = entity.polyline?.positions?.getValue() as Cartesian3[];
```

pass undefined or null for getValue. Actually It should be viewer's clock time, but for now we cannot update features by changing time.

Reply:
I fixed.
https://github.com/reearth/reearth-web/pull/421/commits/482074685ea757a1a12461580cb84f6817960afa